### ISSUE TITLE
tab

### ISSUE THREAD

**Issue by sena1989** [sena1989/angularjslogin1#14]
<html>
<style>
body {font-family: Arial;}

/* Style the tab */
.tab {
  overflow: hidden;
  border: 1px solid #ccc;
  background-color: #f1f1f1;
}

/* Style the buttons inside the tab */
.tab button {
  background-color: inherit;
  float: left;
  border: none;
  outline: none;
  cursor: pointer;
  padding: 14px 16px;
  transition: 0.3s;
  font-size: 17px;
}

/* Change background color of buttons on hover */
.tab button:hover {
  background-color: #ddd;
}

/* Create an active/current tablink class */
.tab button.active {
  background-color: #ccc;
}

/* Style the tab content */
.tabcontent {
  display: none;
  padding: 6px 12px;
  border: 1px solid #ccc;
  border-top: none;
}
</style>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
<body>
<div>
<div class="tab">
  <button class="tablinks active" onclick="openCity(event, 'one')">One</button>
  <button class="tablinks" onclick="openCity(event, 'two1')">Two</button>
</div>

<div id="one" class="tabcontent" style="display: block;>
  <form name="myForm" action="one" id="formOne">
  <input type="radio" name="formType" id="cas_radio" value="cas"/>CAS
  <input type="radio" name="formType" id="vip_radio" value="vip"/>VIP
  <input type="radio" name="formType" id="port_radio" value="portfolio"/>PORTFOLIO ID</br>
  parid <input type="text" id="one" name="one"/><br/>
  cardnumber <input type="text" id="two" name="two"/><br/>
  startdate <input type="text" id="three" name="three"/><br/>
  enddate <input type="text" id="four" name="four"/><br/>
  fortfolioid <input type="text" id="five" name="five"/><br/>
  <button type="submit" form="formOne" value="Submit">Submit</button>
  </form>
</div>

<div id="two1" class="tabcontent">
  <form name="myForm2" action="two" id="formTwo">
  <input type="radio" name="formType1" id="cas_radio1" value="cas"/>CAS
  <input type="radio" name="formType1" id="vip_radio1" value="vip"/>VIP
  <input type="radio" name="formType1" id="port_radio1" value="portfolio"/>PORTFOLIO ID</br>
  parid <input type="text" id="one1" name="one"/><br/>
  startdate <input type="text" id="three1" name="three"/><br/>
  enddate <input type="text" id="four1" name="four"/><br/>
  fortfolioid <input type="text" id="five1" name="five"/><br/>
  File <input type="file" name="file1" id="file1" accept="text/plain" /><br/>
  <button type="submit" form="formTwo" value="Submit">Submit</button>
  </from>
</div>
</div>
</body>

<script>
function openCity(evt, cityName) {
  var i, tabcontent, tablinks;
  tabcontent = document.getElementsByClassName("tabcontent");
  for (i = 0; i < tabcontent.length; i++) {
    tabcontent[i].style.display = "none";
  }
  tablinks = document.getElementsByClassName("tablinks");
  for (i = 0; i < tablinks.length; i++) {
    tablinks[i].className = tablinks[i].className.replace(" active", "");
  }
  document.getElementById(cityName).style.display = "block";
  evt.currentTarget.className += " active";
}

$(document).ready(function() {
    $('input:radio[name=formType]').change(function() {
        if (this.value == 'portfolio') {
            $('#five').val("");
			$('#five').attr("readonly","false");
        }
        else {
            $('#five').val(this.value);
			$('#five').attr("readonly","false");
        }
    });
	
	$('input:radio[name=formType1]').change(function() {
        if (this.value == 'portfolio') {
            $('#five1').val("");
			$('#five1').attr("readonly","false");
        }
        else {
            $('#five1').val(this.value);
			$('#five1').attr("readonly","false");
        }
    });
});

var isFileValid = false;
document.getElementById('file1').onchange =function(){
var regex =/^[0-9\s]*$/;
var file = this.files[0];
var reader = new FileReader();
reader.onload = function(progressEvent){
var lines = this.result.split('\n');
if(lines.length <= 1000){
for(var line =0;line < lines.length;line++){
console.log(lines[line]);
var currentValue = lines[line].toString();
if(regex.test(currentValue) && lines[line].length > 1){
isFileValid = true;
}else{
document.getElementById('file1').value="";
isFileValid = false;
break;
}
console.log(isFileValid);
}
} else {
alert('File count exceeds the limit');
document.getElementById('file1').value="";
}
};
reader.readAsText(file);
};

$('#formTwo').submit(function () {  
   if (isFileValid  == false) {
        alert('File is not valid');
        return false;
    }
});

</script>
</html>
